apps/src/templates/lessonOverview/activities/LevelDetailsDialog.jsx

```diff
@@ -74,7 +83,41 @@ export default class LevelDetailsDialog extends Component {
         </div>
       );
     } else {
-      return <div>Not implemented yet</div>;
+      // TODO: calculate more of these parameters based on the level and pages
+      return (
+        <UnconnectedTopInstructions
+          hasContainedLevels={false}
```

Comment:
Is contained levels a follow up work item?

Reply:
Yes! Contained levels, blockly-defined levels, and anything else I missed will be handled in a follow-up.